host/frontend/liboperator/operator/operator.go

```diff
@@ -97,9 +99,45 @@ func CreateHttpHandlers(
 	router.HandleFunc("/infra_config", func(w http.ResponseWriter, r *http.Request) {
 		ReplyJSONOK(w, config)
 	}).Methods("GET")
+	remoteUrl, proxy, err := getOpenWrtProxy()
+	if err == nil {
+		router.HandleFunc("/openwrt", forwardWithoutUrl(remoteUrl, proxy))
+		// We cannot set up the base url for luci, so just add necessary paths as well.
+		router.HandleFunc("/cgi-bin/{rest:.*}", forwardWithPreservedUrl(remoteUrl, proxy))
+		router.HandleFunc("/luci-static/{rest:.*}", forwardWithPreservedUrl(remoteUrl, proxy))
+	}
```

Comment:
The proxying to operwrt should be parameterized in the library and allow the library users to decide whether to enable it or not.

Reply:
I think it would be exposed regardless of an operator mode. (that page is already accessible in cuttlefish instance)

Reply:
The current two library users are not the only ones. We may use this library from the orchestration project too where this is not necessary..